### ISSUE TITLE
binanceus: Exchange does not support dynamic whitelist in this configuration

### ISSUE THREAD

**Issue by grainz** [freqtrade/freqtrade#8058]
## Describe your environment

  * Operating system: Linux 5.10.0-20-amd64 # 1 SMP Debian 5.10.158-2 (2022-12-13) x86_64 GNU/Linux
  * Python Version: Python 3.9.2
  * CCXT version: ccxt==2.6.65
  * Freqtrade Version: freqtrade 2023.1.dev-6c0fa0dc1

## Describe the problem:

`Exchange does not support dynamic whitelist in this configuration. Please edit your config and either remove Volumepairlist, or switch to using candles. and restart the bot.`

### Steps to reproduce:

Run any configuration on Binance.US using Volumepairlist
  
### Observed Results:

This ran fine before CCXT was updated to include this change: https://github.com/ccxt/ccxt/commit/98e9d00b4e840789ad9f91bf8f06492bb5aa85a9#diff-11bbb38790d9f347079c6051aca341b0f99e4bda47ba6743bb860592176fe0d9

This is not primarily a FT issue however Volumepairlist does rely on this piece of code to run:
VolumePairList.py
```
        if (not self._use_range and not (
                self._exchange.exchange_has('fetchTickers')
                and self._exchange.get_option("tickers_have_quoteVolume"))):
            raise OperationalException(
                "Exchange does not support dynamic whitelist in this configuration. "
                "Please edit your config and either remove Volumepairlist, "
                "or switch to using candles. and restart the bot."
            )
```

CCXT removed fetchTickers for binanceus for some reason.
If I manually enable it in CCXT, the bot runs without issue.
There are "binanceus.py" files that have this line:
`'fetchTickers': False,`
Changing it to True allows FT to run normally.

This is more for visibility on the issue since I don't think FT can do much to resolve this.

### Relevant code exceptions or logs

  ```
2023-01-24 05:38:16,334 - freqtrade.resolvers.iresolver - INFO - Using resolved pairlist VolumePairList from '/home/grainz/build/freqtrade/freqtrade/plugins/pairlist/VolumePairList.py'...
2023-01-24 05:38:16,334 - freqtrade.commands.trade_commands - ERROR - Exchange does not support dynamic whitelist in this configuration. Please edit your config and either remove Volumepairlist, or switch to using candles. and restart the bot.
2023-01-24 05:38:16,334 - freqtrade.commands.trade_commands - ERROR - Fatal exception!
Traceback (most recent call last):
  File "/home/grainz/build/freqtrade/freqtrade/commands/trade_commands.py", line 18, in start_trading
    worker = Worker(args)
  File "/home/grainz/build/freqtrade/freqtrade/worker.py", line 37, in __init__
    self._init(False)
  File "/home/grainz/build/freqtrade/freqtrade/worker.py", line 53, in _init
    self.freqtrade = FreqtradeBot(self._config)
  File "/home/grainz/build/freqtrade/freqtrade/freqtradebot.py", line 77, in __init__
    self.pairlists = PairListManager(self.exchange, self.config)
  File "/home/grainz/build/freqtrade/freqtrade/plugins/pairlistmanager.py", line 36, in __init__
    pairlist_handler = PairListResolver.load_pairlist(
  File "/home/grainz/build/freqtrade/freqtrade/resolvers/pairlist_resolver.py", line 39, in load_pairlist
    return PairListResolver.load_object(pairlist_name, config,
  File "/home/grainz/build/freqtrade/freqtrade/resolvers/iresolver.py", line 182, in load_object
    found_object = cls._load_object(paths=abs_paths, object_name=object_name,
  File "/home/grainz/build/freqtrade/freqtrade/resolvers/iresolver.py", line 156, in _load_object
    return module(**kwargs)
  File "/home/grainz/build/freqtrade/freqtrade/plugins/pairlist/VolumePairList.py", line 78, in __init__
    raise OperationalException(
freqtrade.exceptions.OperationalException: Exchange does not support dynamic whitelist in this configuration. Please edit your config and either remove Volumepairlist, or switch to using candles. and restart the bot.  ```

**Comment by wesgurn** [comment]
I can confirm this issue as well. Thanks for sharing the fix and opening https://github.com/ccxt/ccxt/issues/16614

**Comment by xmatthias** [comment]
As you correctly show / point out - it's an upstream problem - not something we can necessariliy help with.

Great that you've opened an upstream issue. We'll then get either the explanation as to why this was disabled (binance.us doesn't offer this endpoint??) - or a reversal of this setting if it was accidental - which would fix the problem for us.

At any rate - you'll still be able to use VolumePairList - but in the [advanced configuration](https://www.freqtrade.io/en/stable/plugins/#volumepairlist-advanced-mode) - which is less performant (but allows for more customizations ...).

**Comment by xmatthias** [comment]
closed with fee7b792e1c5641cbdc7021ab66485986d5c5ff3.
